apps/lib/p5play/p5.play.js

```diff
@@ -3813,7 +3861,7 @@ function SpriteSheet(pInst) {
   this.drawFrame = function(frame_name, x, y, width, height) {
     var frameToDraw;
     if (typeof frame_name === 'number') {
-      frameToDraw = this.frames[frame_name];
+      frameToDraw = this.frames[frame_name].frame;
```

Comment:
@islemaster I've pulled in the most recent code from p5.play with my most recent changes. The only differences are some formatting and this `.frame` line. I retested manually and everything seems good to go. Should I go ahead and merge?